DOCUMENTRETRIEVAL/documentRetrieval/library/documentRetrieval.py

```diff
@@ -276,14 +276,14 @@ def tfidf_sum_weight(probability,  token_frequency, n, idf, word, alpha, origina
     return tfidf_value
 
 
-def tfidf_score_str(tokens,texts,tfidf_function_name,m = 10,*args):
+def tfidf_score_str(tokens,texts,tfidf_function_name,nb_all_texts_in_db, m = 10,*args):
     """Takes function name as input, returns function"""
     if tfidf_function_name == 'tfidf_sum':
-            return tfidf_score(tokens,texts,tfidf_sum,m)
+            return tfidf_score(tokens,texts,tfidf_sum,nb_all_texts_in_db,m)
     else:
         raise Exception("Error, different function name")
 
-def tfidf_score(tokens, texts, tfidf_function,m = 10, *args):
+def tfidf_score(tokens, texts, tfidf_function,nb_all_texts_in_db,m = 10, *args):
```

Comment:
Missing space between `nb_all_texts_in_db` and `m = 10`.

There should be no spaces when setting default values: `m = 10` -> `m=10`